### PR TITLE
[DA-2257] GEM to GP export script

### PR DESCRIPTION
**Pull request by j-kanuch**
## Description of changes/additions
This PR creates a tool to load the new table `gem_to_gp_migration` from participant informing loop responses. This will be used to provide Color with migration files for the GEM to GP migration alpha testing through production phases.

### CODE REVIEW

File: rdr_service/tools/tool_libs/genomic_utils.py [hunk 1]
```diff
@@ -2000,6 +2003,86 @@ def run(self):
...
+
+        for row in results:
+            obj = self.gem_gp_dao.prepare_obj(row, controller.job_run, file_path)
+
+            batch.append(obj)
+
+            # write to table in batches
+            if len(batch) % batch_size == 0:
+                if not self.args.dryrun:
+                    print(f'Inserting batch starting with: {batch[0].participantId}')
```
Comment (dtharpeuno) [inline rdr_service/tools/tool_libs/genomic_utils.py]: :nit Can we chaneg these to `_logger.info` statements.

File: rdr_service/tools/tool_libs/genomic_utils.py [hunk 2]
```diff
@@ -2000,6 +2003,86 @@ def run(self):
...
+        with GenomicJobController(GenomicJob.GEM_GP_MIGRATION_EXPORT,
+                                  bq_project_id=self.gcp_env.project) as controller:
+
+            results = self.get_data_for_export(controller)
+
+            self.export_to_gem_gp_table(controller, results)
+
+        return 0
+
+    def get_data_for_export(self, controller):
```
Comment (dtharpeuno) [inline rdr_service/tools/tool_libs/genomic_utils.py]: Can we put this query in the `dao`?

**Review by dtharpeuno — APPROVED** [review]

File: rdr_service/tools/tool_libs/genomic_utils.py (hunk 1 above)
Comment (j-kanuch) [reply rdr_service/tools/tool_libs/genomic_utils.py]: Yep, changed it.

File: rdr_service/tools/tool_libs/genomic_utils.py (hunk 2 above)
Comment (j-kanuch) [reply rdr_service/tools/tool_libs/genomic_utils.py]: Good call, moved it.

**Review by dtharpeuno — APPROVED** [review]

**Review by wangy70 — APPROVED** [review]

**Pull request merged**